### ISSUE TITLE
Path to GPG key has changed, causing failure

### ISSUE THREAD

**Issue by TravisWhitehead** [linuxhq/ansible-role-ius#3]
It seems that in the CentOS 7 package from https://centos7.iuscommunity.org/ius-release.rpm the path to the GPG key has changed:
```
[root@vagrant-multi1 ~]# rpm -qlp ./ius-release.rpm
warning: ./ius-release.rpm: Header V4 RSA/SHA256 Signature, key ID 4b274df2: NOKEY
/etc/pki/rpm-gpg/RPM-GPG-KEY-IUS-7
/etc/yum.repos.d/ius-archive.repo
/etc/yum.repos.d/ius-testing.repo
/etc/yum.repos.d/ius.repo
/usr/share/licenses/ius-release-2
/usr/share/licenses/ius-release-2/LICENSE
```

This causes the role to fail to run:
```
TASK [linuxhq.ius : Ensure that the ius gpg keys are installed] ****************
fatal: [vagrant-client.foo.bar]: FAILED! => {"changed": false, "msg": "Not a valid key /etc/pki/rpm-gpg/IUS-COMMUNITY-GPG-KEY"}
```

**Comment by tkimball83** [comment]
@TravisWhitehead - Please test tag/version v0.0.5 and confirm all looks good.

Thanks!

**Comment by TravisWhitehead** [comment]
@tkimball83 I appreciate the quick response. That solves the problem, but I think you may want to account for when installing to CentOS/RHEL 6, as then the key will be named `/etc/pki/rpm-gpg/RPM-GPG-KEY-IUS-6`

**Comment by tkimball83** [comment]
Whoops, that totally slipped by me!  Should be fixed in v0.0.6.

**Comment by TravisWhitehead** [comment]
Thanks!
